lib/tasks.js

```diff
@@ -22,16 +21,14 @@ export function newEmail(emailFrom, to, subject, content) {
 /**
  * Builds an HTML template for the mail to be sent.
  * @param {String} targetEenheidLabel
- * @param {Array | String} submissionUrls
+ * @param {Array} submissionUrls
  * @returns html email template
  */
 export function generateHtmlEmail(targetEenheidLabel, submissionUrls) {
-  const htmlSubmissionLink = Array.isArray(submissionUrls) && submissionUrls.length > 1
-  ? submissionUrls.map(submission => (
-      `<p><a href="${submission}" target="_blank" style="letter-spacing:normal;line-height:100%;text-align:center;text-decoration:none;color:#ffffff;background-color:#1069C9;padding:15px 20px;font-size:16px;border:none;cursor:pointer; font-family:sans-serif;">Bekijk nieuw inzending</a></p><br /><br />`
-    )).join('')
-  : `<p><a href="${submissionUrls}" target="_blank" style="letter-spacing:normal;line-height:100%;text-align:center;text-decoration:none;color:#ffffff;background-color:#1069C9;padding:15px 20px;font-size:16px;border:none;cursor:pointer; font-family:sans-serif;">Bekijk nieuw inzending</a></p><br /><br />`;
 
+  const htmlSubmissionLink = submissionUrls.map(submission => (
+      `<p><a href="${submission}" target="_blank" style="letter-spacing:normal;line-height:100%;text-align:center;text-decoration:none;color:#ffffff;background-color:#1069C9;padding:15px 20px;font-size:16px;border:none;cursor:pointer; font-family:sans-serif;">Bekijk nieuw inzending</a></p><br /><br />`
```

Comment:
Small remark:
"Bekijk nieuw**e** inzending"

Also, I'm not fully up-to-date with the background context here, but I found it a bit strange that the mail shows 3 buttons with the same label, but different links. Do we have any other data available to differentiate the buttons a bit?

Reply:
I guess we can extract the submission type so it will look like this Bekijk nieuwe { submissionType } inzending ?

Reply:
> I guess we can extract the submission type so it will look like this Bekijk nieuwe { submissionType } inzending ?

After rethinking about this, we're likely to have the same problem if the submission types are identical :thinking: